### PR TITLE
Update opt-out USSD menus

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review please

File: go-app-optout.js [hunk 1]
```diff
@@ -857,14 +863,47 @@ go.app = function() {
                     return self.im.contacts
                         .save(self.contact)
                         .then(function() {
-                            return 'states_end';
+                            //TODO: run unsub
+                            if (['not_useful', 'other'].indexOf(choice.value) !== -1){
```
Comment (smn) [inline go-app-optout.js]: [`_.contains()`](http://devdocs.io/lodash/index#contains) may be useful here

File: go-app-optout.js [hunk 2]
```diff
@@ -857,14 +863,47 @@ go.app = function() {
                     return self.im.contacts
                         .save(self.contact)
                         .then(function() {
-                            return 'states_end';
+                            //TODO: run unsub
```
Comment (smn) [inline go-app-optout.js:867]: TODO still relevant?

File: go-app-optout.js [hunk 3]
```diff
@@ -874,6 +913,15 @@ go.app = function() {
             });
         });
 
+        self.states.add('states_end_yes', function(name) {
+            return new EndState(name, {
+                text: $('Thank you. You will receive support messages ' +
+                            'from MomConnect in the coming weeks.'),
```
Comment (smn) [inline go-app-optout.js:920]: How are these messages being sent and how are we tracking who needs to send these?

File: go-app-optout.js (hunk 1 above)
Comment (imsickofmaps) [inline go-app-optout.js]: ugh. I checked the lodash docs under the array section... thanks.

File: go-app-optout.js (hunk 2 above)
Comment (imsickofmaps) [inline go-app-optout.js:867]: Yeah - its a general TODO

File: go-app-optout.js (hunk 3 above)
Comment (imsickofmaps) [inline go-app-optout.js:920]: See line 80 of `src/optout.js`

Comment (imsickofmaps) [inline go-app-optout.js:920]: It's another protocol in the ndoh-control scheduled sender app

**Comment by imsickofmaps** [comment]
@smn can I land?

**Comment by smn** [comment]
:+1:

**Pull request merged**